backend/app/routes/admin/changePassword.js

```diff
@@ -0,0 +1,63 @@
+const to = require('await-to-js').default;
+const argon2 = require('argon2');
+const Admin = require('../../models/Admin');
+const { ErrorHandler } = require('../../../helpers/error');
+const constants = require('../../../constants');
+
+module.exports = async (req, res, next) => {
+  const { oldPassword, newPassword } = req.body;
+
+  const userRecord = await Admin.findOne({
+    email: res.locals.decode.email,
+  });
+
+  const isPasswordCorrect = await argon2.verify(
+    userRecord.passwordHash,
+    oldPassword
+  );
+
+  if (!isPasswordCorrect) {
+    const error = new ErrorHandler(constants.ERRORS.INPUT, {
+      statusCode: 400,
+      message: 'Wrong old password',
+      user: userRecord.email,
+      errStack: 'Password change',
+    });
+    return next(error);
+  }
+
+  if (oldPassword === newPassword) {
+    const error = new ErrorHandler(constants.ERRORS.INPUT, {
+      statusCode: 400,
+      message: 'Old password cannot be same as new password.',
```

Comment:
`new password cannot be identical to old password`

Reply:
@jackfrost13 so this check is not required?

Reply:
I meant the message
`message: 'Old password cannot be same as new password.' `to
`message: 'new password cannot be identical to old password'`

Reply:
ohh got it thank you